app/src/components/mapsMarker.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import Logo from '../images/logolu.png'
+
+function MapsMarker() {
+    return (
+            <img src={Logo} alt="logo of influencer research" className="md:w-6 md:h-8 md:mr-6 md:mb-6 usm:w-6 usm:h-8"></img>
```

Comment:
If the tag won't have any content. You could self-close 
`<img src={Logo} alt="logo of influencer research" className="md:w-6 md:h-8 md:mr-6 md:mb-6 usm:w-6 usm:h-8"/>`